src/SellScene.js

```diff
@@ -5,7 +5,11 @@ import {DEV, retrieveAddress, convertToMillionsUnits, formatAmount} from './util
 import * as API from './api'
 import BuySellForm from './BuySellForm'
 
-const formatResponse = async (res, wallet, valueType, value, cryptoCode, fiatCode) => {
+const formatResponse = async (response, wallet, valueType, value, cryptoCode, fiatCode) => {
+  const {res, err} = response
+  if (err) {
```

Comment:
`const {res, err} = response` will not work since res is no longer an argument, it has been renamed to response

Reply:
Inside response there is a res or error.

Reply:
Gotchya. ok.